### PR TITLE
Modified the WME monitoring docs

### PR DESCRIPTION
**Pull request by saikrishnagobburi**

### CODE REVIEW

File: learn/on-premise/observability/logs-aggregation/kibana.md [hunk 1]
```diff
@@ -52,6 +52,20 @@ Kibana is purely used for the logs generated by the Platform. Application logs g
 
     ![Kibana Discover Page](/learn/assets/wme-setup/wme-observability/kibana/kibana-discover-open.png)
 
+    **UseCaseOne**: Accessing Specific MicroService log( Developer-Cloud) from Kibana.
```
Comment (krishnaraoch) [inline learn/on-premise/observability/logs-aggregation/kibana.md:55]: Add this end of the doc. not in the middle.
Heading can be like Usecases. In that numbers/bullets..
Usecases
1. Accessing Specific MicroService Logs.
    There are micro services like login, developer-cloud.. write all micro service names.
    screen shot is wrong. it should be with tag not with log file name.
    
2.Access All MicroService Logs
   I didnt see dashboard for this. I assume u seen it in one of the wme setup

File: learn/on-premise/observability/metrics-collection/alerts.md [hunk 2]
```diff
@@ -85,3 +85,13 @@ Following are the list of alerts included in WaveMaker
 :::note
 The Alerts listed above keep changing as new things are discovered and might vary over time
 :::
+
+### Configuring Alerts
```
Comment (krishnaraoch) [inline learn/on-premise/observability/metrics-collection/alerts.md:89]: Heading : Configure Notifications for Alerts.
https://prometheus.io/docs/alerting/latest/notification_examples/

We are allowing our customers are configure notifications to various channel.  EMail is once channel.
there are other channel as listed in thier website.  So you understand from above link and write it down.

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

File: learn/on-premise/observability/logs-aggregation/kibana.md (hunk 1 above)
Comment (saikrishnagobburi) [reply learn/on-premise/observability/logs-aggregation/kibana.md:55]: 2) I have seen Dashboard in the demo-setup and also in our exsi setup, based on that i have added

Comment (saikrishnagobburi) [reply learn/on-premise/observability/logs-aggregation/kibana.md:55]: Added the tag.keyword filter made the changes

File: learn/on-premise/observability/metrics-collection/alerts.md [hunk 3]
```diff
@@ -85,3 +85,17 @@ Following are the list of alerts included in WaveMaker
 :::note
 The Alerts listed above keep changing as new things are discovered and might vary over time
 :::
+
+### Configure Notifications for Alerts
+Prometheus uses alert manager to send notifications.In WaveMaker WME Setup alerts are configured through Email channel. Below link explains configuration of alerts using various channels like Slack ,Pageduty etc.
```
Comment (krishnaraoch) [inline learn/on-premise/observability/metrics-collection/alerts.md:90]: Eitther call it as WaveMaker Enterprise Setup or WaveMaker OnPremise Setup

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

**Pull request closed without merging**